src/index.ts

```diff
@@ -0,0 +1,11 @@
+export class Challenge 
+{
+    buzz(num: number): Promise<string | number>
+    {
+        let res: string | number = "";
+        if (num % 3 === 0) res += "fizz";
```

Comment:
Add a space before an if statement.
Remove space between if(
res must stay in a new line.
This kind of issue will be expressed as 🙈